File: RestApi/RestApiMinority/RestApiMinority/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RestApiMinority.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Mail { get; set; }
        public string Password{ get; set; }
        public int Monedas{ get; set; }
        public int SalaDeJuego{ get; set; }

        public void LlenarDatosCon0()
        {
            Id = 0;
            Nombre = "0";
            Mail = "0";
            Password = "0";
            SalaDeJuego = 0;
        }
    }
}

File: App/Minority/Minority/app/src/main/java/com/revich/mobile/minority/Usuario.java
package com.revich.mobile.minority;

/**
 * Created by Igal on 6/13/2017.
 */

public class Usuario
{
    public int Id;
    public String Nombre;
    public String Mail;
    public String Password;
    public int Monedas;
    public int SalaDeJuego;

    public void LlenarDatos(int Id, int Monedas, int SalaDeJuego )
    {
        this.Id=Id;
        this.Nombre="Igal";
        this.Mail="Igal";
        this.Password="igal";
        this.Monedas=Monedas;
        this.SalaDeJuego= SalaDeJuego;
    }
}


